server/auth.go

```diff
@@ -447,8 +451,80 @@ func (s *Server) processClientOrLeafAuthentication(c *client, opts *Options) boo
 	if s.trustedKeys != nil {
 		if c.opts.JWT == _EMPTY_ {
 			s.mu.Unlock()
-			c.Debugf("Authentication requires a user JWT")
-			return false
+
+			var acc *Account
+			if c.opts.Account == _EMPTY_ {
+				// check default (system account) callout
+				acc = s.SystemAccount()
+				if acc == nil {
+					c.Debugf("Authentication requires a user JWT - Token Request failed - Account Missing")
+					return false
+				}
+			} else {
+				// check account listed
+				acc, err = s.fetchAccount(c.opts.Account)
+				if err != nil {
+					c.Debugf("Authentication requires a user JWT - Token Request Account %q lookup failed: %s", c.opts.Account, err)
+					return false
+				}
+			}
+			acc.mu.RLock()
+			signupName := fmt.Sprintf("Account: %s/%s", acc.Name, acc.nameTag)
+			featOnboard := acc.tags.Contains(featureTagOnboard)
+			acc.mu.RUnlock()
+			if !featOnboard {
+				c.Debugf("Authentication requires a user JWT")
+				return false
+			}
+			// Perform the actual JWT Request
+			msgChan := make(chan []byte)
+			defer close(msgChan)
+			ib := s.newRespInbox()
+			sub, err := s.sysSubscribe(ib, func(_ *subscription, _ *client, acc *Account, subject, reply string, rmsg []byte) {
+				msgChan <- copyBytes(rmsg)
```

Comment:
I see that later you convert to a string anyway, so I would have made `msgChan` a chan string instead and here have `msgChan <- string(rmsg)`.

Reply:
fixed